Aircraft/JA37/Nasal/payload/missiles.nas

```diff
@@ -5025,7 +5025,7 @@ var AIM = {
 	
 	steering_speed_G: func(meHeading, mePitch, steering_e_deg, steering_h_deg, s_fps, dt) {
 		if (s_fps == 0) {
-			return 1;
```

Comment:
If the missile is still, it has 1 G (at least if its level)

Reply:
Not that it matters, if it has 0 speed, then something went wrong..

Reply:
My reasoning is that if the speed is 0, it should be free falling (so 0G), unless the missile is levitating by black magic.
Also, the function is (in this version) linear in `s_fps`, so should return 0 if `s_fps` is 0.

Reply:
Well speed in this case is total speed, including that from free falling.

Reply:
Do you want me to just delete this test? (the rest of the function also works with `s_fps = 0`)